EIPS/eip-template.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
```suggestion
---
eip: 8116
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.